spec/acceptance/server_spec.rb

```diff
@@ -2,6 +2,10 @@
 
 describe 'hyperglass::server class' do
   context 'default parameters' do
+    # This will allow hiera to lookup test data as hyperglass::server::devices
+    # must be populated for the service to start.
+    on(hosts, 'mkdir -p /etc/facter/facts.d && echo -e "---\nbeaker: true" > /etc/facter/facts.d/beaker.yaml')
```

Comment:
Btw, I like this pattern and want to copy this

Reply:
Thanks! Hopefully we don't need it too often. Cleanest I could think of as opposed to adding a bunch of stuff to the spec_helper_acceptance.

Reply:
Please have a look at https://github.com/voxpupuli/voxpupuli-acceptance/pull/13. Currently it's mostly untested, but I think it mostly captures my thoughts on a generic approach.